### PR TITLE
Fixed translations for Subway Exit

### PR DESCRIPTION
**Pull request by biodranik**
Thanks to @FloEdelmann in  #3725

### CODE REVIEW

File: iphone/Maps/LocalizedStrings/es-MX.lproj/Localizable.strings [hunk 1]
```diff
@@ -1866,22 +1866,22 @@
 
 "type.highway.footway" = "Camino";
 
-"type.highway.footway.alpine_hiking" = "Path";
+"type.highway.footway.alpine_hiking" = "Camino";
```
Comment (FloEdelmann) [inline iphone/Maps/LocalizedStrings/es-MX.lproj/Localizable.strings:1869]: The changes in this file seem unrelated.

**Review by FloEdelmann — APPROVED** [review]
Apart from some unrelated changes, this looks good, thanks!

**Comment by FloEdelmann** [comment]
The French translation is also a verb ("Quitter"), the correct French translation should be "Sortie".

File: iphone/Maps/LocalizedStrings/es-MX.lproj/Localizable.strings (hunk 1 above)
Comment (biodranik) [reply iphone/Maps/LocalizedStrings/es-MX.lproj/Localizable.strings:1870]: That's ok, translations were not updated properly before.

**Comment by biodranik** [comment]
@FloEdelmann Thanks a lot, good catch!

**Review by Kazing — APPROVED** [review]

**Review by jimcarst — CHANGES_REQUESTED** [review]
Why is my review needed? It would be easier to review if the strings aren't regenerated, so you only see `strings.txt` in the commit.

For nl, `core_exit` should be: “Uitgang”.

File: data/strings/strings.txt [hunk 2]
```diff
@@ -16433,21 +16434,21 @@
...
     id = Keluar
     it = Uscita
     ja = 終了
     ko = 끝내기
     mr = बाहेर पडा
     nb = Avslutt
-    nl = Verlaten
+    nl = Uitgang
     pl = Wyjdź
     pt = Sair
```
Comment (matheusgomesms) [inline data/strings/strings.txt:16453]: Saída

File: data/strings/strings.txt [hunk 3]
```diff
@@ -16433,21 +16434,21 @@
...
     pt-BR = Sair
```
Comment (matheusgomesms) [inline data/strings/strings.txt:16454]: Saída

**Review by matheusgomesms — CHANGES_REQUESTED** [review]
PT and PT-BR: Saída

(Sorry, I can't edit the file right now, so I just left a comment)

**Review by matheusgomesms — APPROVED** [review]

**Review by lunarna-gh — CHANGES_REQUESTED** [review]
PL translation is also a verb, should be "Wyjście"

**Review by lunarna-gh — APPROVED** [review]

**Review by vng — APPROVED** [review]

**Comment by FloEdelmann** [comment]
Thanks, great to see this fixed in many languages! :blush:
